src/cpsr.rs

```diff
@@ -7,7 +7,7 @@ use crate::{bitwise::Bits, condition::Condition};
 pub struct Cpsr(u32);
 
 impl Cpsr {
-    pub(crate) fn can_execute(&self, cond: Condition) -> bool {
+    pub(crate) fn can_execute(&self, cond: &Condition) -> bool {
```

Comment:
Probably if we add `Copy` trait there's no need to add reference here, but no blocker from my side.